src/css/base/_mixins.scss

```diff
@@ -150,6 +150,12 @@
 
   Synchronize changes with src/js/helpers/reactMediaQueries.js
 **/
+@mixin mobile-small {
+  @media (max-width: #{$coa-small-screen - 1px}) {
+    @content;
+  }
+}
+
```

Comment:
@briaguya here's where I added the `mobile-small`. Let me know if this doesn't look right?

Reply:
looks like `#coa-small-screen`is 480px. Would that make it so we have 4 different responsive breaks for this?

* 0 - 479px wide - mobile-small
* 480 - 768px wide - mobile
* 768 - 1079px wide - tablet
* 1080px+ wide - desktop

Is that the in

Reply:
Well there's actually nothing for tablet. So three-across-four

Reply:
so our responsive breaks for this page are
* 0 - 479px - mobile small
* 480 - 1080px - mobile/tablet (I don't think we have a mixin for this one yet, our current mobile-and-tablet mixin is actually just "not desktop", this is really more of a nice-to-have than anything)
* 1080px+ desktop

Reply:
@briaguya, Ok, so, I had to double-check, but here's what it is. 
- 0 - 479 `@mixin mobile-small` via $coa-small-screen`
- 480 - 767 `@mixin mobile` via `$coa-medium-screen`
- 780+ is just default - I didn't specify any size. 

I thin what tripped me up was that `$coa-medium-screen` is used for the `mobile` in mixin.

Reply:
We have designs for a screen smaller than 480px? That is super small. Like smaller than iphone5

Reply:
@Niicck Most phone are under 480px, even the iphone Max is 414px. But, a lot of sites will use pixel-doubling, which tells phones to render 1:2. Making 414px screen appear as a 828px desktop screen. But, we tell all devices to render 1:1. That's set in our `meta` tag. 

Here's how it looks currently,
`<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=5, shrink-to-fit=no">`

Reply:
Today I learned. Thank you!